source/class/qx/tool/compiler/Version.js

```diff
@@ -28,7 +28,11 @@ qx.Class.define("qx.tool.compiler.Version", {
   },
 
   defer: function(statics) {
-    var pkg = require("../../../../../package.json");
-    statics.VERSION = pkg.version;
+    try {
+      var pkg = require("../../../../../package.json");
+      statics.VERSION = pkg.version;
+    } catch (e) {
+      statics.VERSION = qx.core.Environment.get("qx.compilerVersion");
```

Comment:
maybe better call it `qx.compiler.version`? to avoid camelCase in config key names?